src/component/feature/FeatureView/FeatureMetrics/FeatureMetrics.tsx

```diff
@@ -1,50 +1,135 @@
-                    {metricComponents}
-                </div>
-            </PageContent>
-
-            <PageContent
-                headerContent="Applications"
-                className={styles.secondaryContent}
+        <PageContent headerContent="">
+            <Grid
+                container
+                component="header"
+                spacing={2}
+                alignItems="flex-end"
             >
-                <div className={styles.applicationsContainer}>
-                    <FeatureSeenApplications />
-                </div>
-            </PageContent>
-        </>
+                <Grid item xs={12} md={5}>
+                    {environments.size > 0 && (
+                        <FeatureMetricsChips
+                            title="Environments"
+                            values={environments}
+                            value={environment}
+                            setValue={setEnvironment}
+                        />
+                    )}
+                </Grid>
+                <Grid item xs={12} md={5}>
+                    {applications.size > 0 && (
+                        <FeatureMetricsChips
+                            title="Applications"
+                            values={applications}
+                            value={application}
+                            setValue={setApplication}
+                        />
+                    )}
+                </Grid>
+                <Grid item xs={12} md={2}>
+                    <MobileMarginTop>
+                        <FeatureMetricsHours
+                            hoursBack={hoursBack}
+                            setHoursBack={setHoursBack}
+                        />
+                    </MobileMarginTop>
+                </Grid>
+            </Grid>
+            <FeatureMetricsContent
+                metrics={filteredMetrics}
+                hoursBack={hoursBack}
+            />
+        </PageContent>
+    );
+};
+
+// Get all the environment names for a feature,
+// not just the one's we have metrics for.
+const useFeatureMetricsEnvironments = (
+    projectId: string,
+    featureId: string
+): Set<string> => {
+    const { feature } = useFeature(projectId, featureId);
+
+    const environments = feature.environments.map(environment => {
+        return environment.name;
+    });
+
+    return new Set(environments);
+};
+
+// Get all application names for a feature. Fetch apps for the max time range
+// so that the list of apps doesn't change when selecting a shorter range.
+const useFeatureMetricsApplications = (featureId: string): Set<string> => {
+    const { featureMetrics = [] } = useFeatureMetricsRaw(
+        featureId,
+        FEATURE_METRIC_HOURS_BACK_MAX
     );
+
+    const applications = featureMetrics.map(metric => {
+        return metric.appName;
+    });
+
+    return new Set(applications);
 };
 
+const MobileMarginTop = styled(Box)(({ theme }) => ({
```

Comment:
Any benefit to defining useStyles here? This would be the only place we use styled components.